src/backends/tensorrt/tensorrtlib.cc

```diff
@@ -196,6 +199,22 @@ namespace dd
             + this->_mlmodel._repo);
       }
 
+    _top_k = ad.has("topk") ? ad.get("topk").get<int>() : 200;
```

Comment:
This is the line I'd recommend we have set the default value for top k somewhere else. Though I don't want to block the PR for this if it can't be fixed now.
Can you add a comment that this needs to go elsewhere ?

Reply:
Done